internal/api/v1/namespace/create.go

```diff
@@ -87,7 +87,8 @@ func addNamespaceToUser(ctx context.Context, namespace string) error {
 
 	err = authService.AddNamespaceToUser(ctx, user.Username, namespace)
 	if err != nil {
-		return errors.Wrap(err, fmt.Sprintf("error adding namespace [%s] to user [%s]", namespace, user.Username))
+		return errors.Wrap(err, fmt.Sprintf("error adding namespace [%s] to user [%s]",
```

Comment:
Same for `Wrapf`

Reply:
Done with commit f6bc2eca5f02b7f0634afe443b9e2ea734faeb05